src/pages/instances/CreateInstance.tsx

```diff
@@ -578,6 +583,22 @@ const CreateInstance: FC = () => {
           </ActionButton>
         )}
       </FormFooterLayout>
+
+      {(panelParams.panel === panels.editNetworkDevice ||
+        panelParams.panel === panels.createNetworkDevice) && (
+        <NetworkDevicePanelContainer
+          mode={isEditNetworkDevice ? "edit" : "create"}
+          deviceName={
+            isEditNetworkDevice && panelParams.deviceName
+              ? panelParams.deviceName
+              : undefined
+          }
+          project={project}
+          formik={formik}
+          onClose={panelParams.clear}
+          onSave={panelParams.clear}
+        />
+      )}
```

Comment:
I think this should stay visible when clicking to another part of the form (say go from networks to Main configuration). currently it unmounts due to the url losing the panel parameters.